src/sass/components/_cards.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  margin-bottom: 30px;
+
+  .card-header {
+    padding: $card-spacer-x;
+    background-color: $primary;
+    color: color-yiq($primary);
+
+    .card-title {
+      margin-bottom: 0;
+
+      h1:last-child,
+      h2:last-child,
+      h3:last-child,
+      h4:last-child,
+      h5:last-child,
+      h6:last-child {
+        margin-bottom: 0;
+      }
+    }
+  }
+
+  .card-footer {
+    background-color: $gray-200;
+  }
+
+  &.card-dashboard {
+    cursor: pointer;
+    position: relative;
+    transition: box-shadow 0.25s;
+
+    &:hover {
+      box-shadow: $box-shadow;
+    }
+
+    &:focus-within {
+      box-shadow: $box-shadow;
+
+      a:focus {
+        text-decoration: none;
+      }
+    }
+
+    i {
+      font-size: 4rem;
+      margin-bottom: 1rem;
+    }
+
+    h2 {
+      font-size: $font-size-base;
+    }
+
+    a {
```

Comment:
Is dit styling voor een anchor in een anchor? U card is ook een anchor lijdt ik af uit u bovenstaande css

Reply:
Nee, de card is opgebouwd volgens https://inclusive-components.design/cards/, waardoor je op de hele card kan klikken maar de link toch enkel op de titel bv ligt.

Reply:
checken link in card waarbij de card niet helemaal aanklikbaar is